dev_tools/profiling/benchmark_serializers.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tool for benchmarking serialization of large circuits."""
```

Comment:
Add more documentation here, since it's a script just chillin' in the `dev_tools` folder and may be confusing when uncovered by future code explorers. Maybe even link to relevant issues and PRs, e.g. this one

Reply:
Done. Added links and a sample run command (requires installing the local version of Cirq, or cirq-dev once it picks up this change).